app/routes/circulars/index.tsx

```diff
@@ -19,6 +19,7 @@ import { getUser } from '../__auth/user.server'
 import { CircularsServer } from './circulars.server'
 
 export async function loader({ request }: DataFunctionArgs) {
+  if (!process.env['FEATURE']) throw new Response('', { status: 404 })
```

Comment:
Please call this feature flag something more descriptive, like `GCN_CIRCULARS_ENABLE`.

Note that we already have a route /circulars on the main branch, which should not 404 error.

Reply:
True, I have updated the file names to keep the /circulars route behaving as it should